app/models/configuration_script.rb

```diff
@@ -1,3 +1,17 @@
 class ConfigurationScript < ActiveRecord::Base
-  belongs_to :configuration_manager, :class_name => 'ManageIQ::Providers::ConfigurationManager'
+  belongs_to  :manager,
+              :class_name  => 'ManageIQ::Providers::ConfigurationManager',
+              :foreign_key => :configuration_manager_id
+  include ProviderObjectMixin
+
+  def run(vars=nil)
+    extra_vars = vars || variables
```

Comment:
The `variables` column contains the default values for the JobTemplate so should we merge it with `vars`?  Is there ever a reason we would **not** want to pass the values defined in `variables`?

```ruby
def run(vars={})
  extra_vars = vars.reverse_merge(variables)
```

@bdunne @bzwei @mkanoor Thoughts?